src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -486,18 +486,19 @@ private async Task EnsureModulesLoaded()
                     {
                         foreach (string file in _libraryLoaded)
                         {
-                            if (_launchOptions.SymbolInfoExceptionList.Contains(file))
+                            string filename = Path.GetFileName(file);
```

Comment:
Path.GetFileName will throw if the specified file name contains any characters which aren't valid file name characters on Windows. You probably want to either catch ArgumentException or make your own helper function that does the same thing but in a less bad way.

Reply:
New PR submitted to address this comment.